docs/shared-content-source/docs/modules/develop/pages/monitoring.adoc

```diff
@@ -0,0 +1,100 @@
+:page-partial:
+
+include::ROOT:partial$include.adoc[]
+
+## Monitoring Akka Streamlets
+
+Cloudflow allows you to use https://developer.lightbend.com/docs/telemetry/current/home.html[telemery] just by adding the necessary configuration. Telemetry provides instrumentation to report metrics for Akka components through different plugins, such as Prometheus or Elasticsearch. These metrics can then be accessed through, for instance, a Graphana dashboard or a Kibana one. 
+
+
+In this case, we'll describe the use case where the reporter is Prometheus and the dashboard is Grafana. Which is the same as in the https://developer.lightbend.com/docs/cloudflow/current/install/install-console.html[Cloudflow Console]
+
+The required configuration is, as per https://developer.lightbend.com/docs/telemetry/current/setup/cinnamon-agent-sbt.html#lightbend-telemetry-sbt-plugin[documentation], the following:
+
+### Add sbt-cinnamon plugin 
+We must add sbt-cinnamon plugin into `plugins.sbt` file, in the `project` folder.  
+
+```
+addSbtPlugin("com.lightbend.cinnamon" % "sbt-cinnamon" % [version])
+```
+changing `version` to https://developer.lightbend.com/docs/telemetry/current/setup/cinnamon-agent-sbt.html#lightbend-telemetry-sbt-plugin[latest]
+
+### Add library dependencies
+
+The required libraries that will be in charge of adding the telemetry for each Akka component plus the reporter of choice. In our sample
```

Comment:
Just refer to prometheus instead of the reporter of choice

Reply:
done